### DISCUSSION TITLE
关于本科毕业论文编译后pdf的查重问题

### DISCUSSION THREAD

**Discussion by MZGuo111** [whutug/whu-thesis#127] [Q&A]
感谢学长学姐们提供的模板！我是一名正在做本科毕设的whuer，我想请问一下编译后的pdf文件可以正常进入学校的查重系统进行查重吗，会识别不出来出现乱码吗？本科查重系统要求的是最好上传word原稿。

**Comment by tanukihee** [comment]
可以正常查重，不会出现乱码

**Reply by MZGuo111** [reply]
感谢解答！
